client.py

```diff
@@ -4,19 +4,34 @@
 from hazelcast.discovery import HazelcastCloudDiscovery
 
 """
-This is boilerplate application that configures client to connect Hazelcast Cloud cluster.
-After successful connection, it puts random entries into the map.
+This is a boilerplate client application that connects to your Hazelcast Viridian cluster.
+See: https://docs.hazelcast.com/cloud/get-started
 
-See: https://docs.hazelcast.cloud/docs/python-client
+Snippets of this code are included as examples in our documentation,
+using the tag:: comments.
 """
 
 logging.basicConfig(level=logging.INFO)
+# tag::env[]
+# Define which environment to use such as production, uat, or dev
 HazelcastCloudDiscovery._CLOUD_URL_BASE = "YOUR_DISCOVERY_URL"
+# end::env[]
+# Configure the client to connect to the cluster
+# tag::config[]
 client = hazelcast.HazelcastClient(
     cluster_name="YOUR_CLUSTER_NAME",
+    # The cluster discovery token is a unique token
+    # that maps to the current IP address of the cluster.
+	# Cluster IP addresses may change.
+	# This token allows clients to find out the current IP address
+	# of the cluster and connect to it.
     cloud_discovery_token="YOUR_CLUSTER_DISCOVERY_TOKEN",
+    # Allow the client to collect metrics
+	# so that you can see client statistics in Management Center.
```

Comment:
Why is this (and some others) are indented with an extra tab?

Reply:
will fix that